src/GovUk.Education.ExploreEducationStatistics.Content.Model/DataSetFileMeta.cs

```diff
@@ -11,31 +12,52 @@ public class DataSetFileMeta
     public List<string> GeographicLevels { get; set; } = new();
```

Comment:
This would probably be better as `List<GeographicLevel>` to provide type safety here.

In the public API model, we also store the level codes (the enum values) in the DB JSON e.g. `NAT`, `LA`, etc, rather than the labels. The codes are not supposed to be changed often, so they are a better choice for this.

This is a more flexible approach that allows us to change the labels we return in the view models, without having to change them in the DB.

Given that we're already running a new migration, would suggest we also update the geographic levels too to store the codes instead.

Reply:
I've made the change. I ended up doing it with Newtonsoft.Json - my rational being that's what we are currently using for all other ContentDbContext JSON stuff so this keeps it consistent.